documentation/release_notes.rst

```diff
@@ -8,6 +8,51 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.9.0
+===============
+
+New Features
+------------
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``fill``, ``move``, ``replace``, ``replace_if``,
+``remove``, ``remove_if``, ``mismatch``, ``minmax_element``, ``min``, ``max``, ``find_first_of``, ``find_end``, ``is_sorted_until``
+- Improved performance of set operation algorithms (e.g., ``set_union``, ``set_difference``, etc.)
```

Comment:
```suggestion
- Improved performance of set operation algorithms: ``set_union``, ``set_difference``, ``set_intersection``, ``set_symmetric_difference``. 
```

There are not many set algorithms. Let's specify them all.

Reply:
There are also set_union_by_key, set_intersection_by_key, set_symmetric_differency_by_key, and set_difference_by_key.  This is too long in my opinion.

Reply:
the by_key algorithms are in the SYCLomatic helper headers, not oneDPL itself, so I don't think we need to include those here.  I dont have a strong opinion about whether to list everything out though.